policyengine_us/parameters/gov/states/vt/tax/income/credits/ctc/reduction/amount.yaml

```diff
@@ -0,0 +1,14 @@
+description: Vermont reduces the child tax credit amount by this amount for each increment of adjusted gross income exceeding the threshold.
+values:
+  2021-01-01: 0
+  2022-01-01: 20
+metadata:
+  unit: currency-USD
+  label: Vermont Child Tax Credit reduction amount
+  reference:
+    - title: Vt. Stat. tit. 32 § 5830f Section (b)
+      href: https://casetext.com/statute/vermont-statutes/title-32-taxation-and-finance/chapter-151-income-taxes/subchapter-002-taxation-of-individuals-trusts-and-estates/section-5830f-see-note-vermont-child-tax-credit/1#page=1
+    - title: Vt. 2022 Tax Form 
+      href: https://tax.vermont.gov/sites/tax/files/documents/Income%20Booklet-2022.pdf#page=24
```

Comment:
this link does not reference the $20

Reply:
https://tax.vermont.gov/sites/tax/files/documents/Income%20Booklet-2022.pdf#page=17